generic/dockerfile/best-practice/missing-zypper-clean.yaml

```diff
@@ -1,9 +1,9 @@
 rules:
   - id: missing-zypper-clean
     severity: WARNING
-    languages: [generic]
+    languages: [dockerfile]
     patterns:
-      - pattern: zypper $COMMAND
+      - pattern: RUN ... zypper $COMMAND ...
```

Comment:
this may cause merge conflicts with https://github.com/returntocorp/semgrep-rules/pull/1926